### PR TITLE
[PREVIEW] Div 3990 cr cya

### PR DESCRIPTION
**Pull request by qzhou-hmcts**
# Description
https://tools.hmcts.net/jira/browse/DIV-3990
-Created CYA for co-respondent
-Added done page without content to complete the page flow.
- Note this changes are applied on the top of DIV-3974-Corresp-Review-App

Fixes # (issue)

- DIV-3990

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: steps/correspondent/cr-done/CrDone.step.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { ExitPoint } = require('@hmcts/one-per-page');
+const config = require('config');
+const idam = require('services/idam');
+
+//  WIP
```
Comment (dtaylor7) [inline steps/correspondent/cr-done/CrDone.step.js]: ?

File: steps/correspondent/cr-check-your-answers/CrCheckYourAnswers.step.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const content = require('./CrCheckYourAnswers.content');
+
+class CrCheckYourAnswers extends CYA {
+  static get path() {
+    return config.paths.correspondent.checkYourAnswers;
+  }
+
+  constructor(...args) {
+    super(...args);
+    this.sendToAPI = this.sendToAPI.bind(this);
```
Comment (dtaylor7) [inline steps/correspondent/cr-check-your-answers/CrCheckYourAnswers.step.js]: makes no sense?

File: steps/correspondent/cr-done/CrDone.step.js (hunk 1 above)
Comment (qzhou-hmcts) [reply steps/correspondent/cr-done/CrDone.step.js]: the done page will be done on https://github.com/hmcts/div-respondent-frontend/pull/230
I added it to complete the 'next' method

File: steps/correspondent/cr-check-your-answers/CrCheckYourAnswers.step.js (hunk 2 above)
Comment (qzhou-hmcts) [reply steps/correspondent/cr-check-your-answers/CrCheckYourAnswers.step.js]: @dtaylor7  I copied from respondent CYA
https://github.com/hmcts/div-respondent-frontend/blob/d39879c564942fd728c462a372deaf3b21b30409/steps/respondent/check-your-answers/CheckYourAnswers.step.js#L17
Do you think is wrong?

File: config/custom-environment-variables.yml [hunk 3]
```diff
@@ -52,9 +52,7 @@ services:
     expire: RATE_LIMITER_EXPIRE
     enabled: RATE_LIMITER_ENABLED
   caseOrchestration:
-    getPetitionUrl: GET_PETITION_URL
-    linkRespondentUrl: LINK_RESPONDENT_URL
-    submitAosUrl: SUBMIT_AOS_URL
+    baseUrl: COS_BASE_URL
```
Comment (hosainnet) [inline config/custom-environment-variables.yml]: Nice 👍

File: config/default.yml [hunk 4]
```diff
@@ -100,12 +98,14 @@ paths:
...
-    confirmDefence: /co-respondent/defend-are-you-sure 
+    confirmDefence: /co-respondent/defend-are-you-sure
     agreeToPayCosts: /co-respondent/agree-to-pay-costs
     contactDetails: /co-respondent/contact-details
+    checkYourAnswers: /co-respondent/check-your-answers
+    done: /co-respondent/done
 tests:
   e2e:
-    show: true
+    show: false
```
Comment (hosainnet) [inline config/default.yml]: I think it's useful to have this one by default. It's off for CI

File: steps/co-respondent/cr-done/CrDone.step.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const { ExitPoint } = require('@hmcts/one-per-page');
+const config = require('config');
+const idam = require('services/idam');
+
+//  WIP
```
Comment (hosainnet) [inline steps/co-respondent/cr-done/CrDone.step.js]: ?

File: steps/co-respondent/cr-done/CrDone.step.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+class CrDone extends ExitPoint {
```
Comment (hosainnet) [inline steps/co-respondent/cr-done/CrDone.step.js]: Do we need a test for this guy

File: test/e2e/paths/co-respondent/adulteryJourney.js [hunk 7]
```diff
@@ -64,4 +68,8 @@ Scenario('Proceed to adultery admission screen and do not admit adultery', I =>
   I.seeCrContactDetailsPage();
   I.consentToSendingCrNotifications();
   I.navByClick(content.en.continue);
+
+  I.seeCrCheckYourAnswersPage();
+  I.confirmInformationIsTrue();
+  I.submitApplication();
```
Comment (hosainnet) [inline test/e2e/paths/co-respondent/adulteryJourney.js]: We don't verify the done page here

File: steps/co-respondent/cr-done/CrDone.step.js (hunk 6 above)
Comment (alexwakeman) [reply steps/co-respondent/cr-done/CrDone.step.js]: Also `CrDone` is a bit cryptic a name, could be `CoRespDone`.

Comment (qzhou-hmcts) [reply steps/co-respondent/cr-done/CrDone.step.js]: This is done on https://github.com/hmcts/div-respondent-frontend/pull/230

File: test/e2e/paths/co-respondent/adulteryJourney.js (hunk 7 above)
Comment (qzhou-hmcts) [reply test/e2e/paths/co-respondent/adulteryJourney.js]: done page is covered on this  PR https://github.com/hmcts/div-respondent-frontend/pull/230 out of the story scope

File: steps/co-respondent/cr-done/CrDone.step.js (hunk 6 above)
Comment (qzhou-hmcts) [reply steps/co-respondent/cr-done/CrDone.step.js]: @alexwakeman  This pattern is followed by all the co-respondent pages

**Comment by qzhou-hmcts** [comment]
This .PR has been merged, is you have comments I'm happy to fix them here https://github.com/hmcts/div-respondent-frontend/pull/237

**Pull request merged**